pkg/operator/ceph/csi/spec.go

```diff
@@ -311,5 +315,45 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 			return fmt.Errorf("failed to create rbd service: %+v\n%+v", err, cephfsService)
 		}
 	}
+
+	if ver.Minor >= provDeploymentSuppVersion {
```

Comment:
Shouldn't this be checking the major version as well?

Reply:
yes that a good idea, will do it